<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPolzEdit</class>
 <widget class="QWidget" name="FPolzEdit">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Пользователь:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_polz"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_pas1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Повтор пароля:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_pas2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_super">
       <property name="text">
        <string>Суперпользователь</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Записать</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/upgrade_downgrade_account.png</normaloff>:/res/1/upgrade_downgrade_account.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/1/list_suspended_accounts.png</normaloff>:/res/1/list_suspended_accounts.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
